db/migrate/20230920171722_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Kindly change the user foreign key attribute from the user (user_id) to the author (author_id), as per project requirements.

__HINT:__  `t.references :author, foreign_key: { to_table: 'users' }`


![data](https://github.com/SAZZAD1Q2/BlogHub/assets/53124548/a0fd1155-598e-45ad-abda-96f85d8c0abb)